src/styles/components/_jca-credit-card.scss

```diff
@@ -0,0 +1,192 @@
+
+    input {
+      width: 100%;
+      background: none;
+      border: none;
+      color: $white;
+      text-align: center;
+      letter-spacing: 2px;
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &__meta {
+    display: flex;
+    color: $white;
+    padding: $default_padding;
+    padding-top: 0;
+  }
+
+  &__owner {
+    flex: 3;
+
+    input {
+      width: 100%;
+      background: none;
+      border: none;
+      color: $white;
+      text-transform: uppercase;
+      padding: 0;
+      margin: 0;
+      font-weight: 700;
+    }
+  }
+  
+  &__expiry {
+    flex: 1;
+    text-align: right;
+  }
+
+  &__month {
+    display: inline-block;
+    border: none;
+    width: 30px;
+    text-align: right;
+    background: none;
+    color: $white;
+    font-weight: 300;
+  }
+
+  &__year {
+    display: inline-block;
+    border: none;
+    width: 40px;
+    text-align: left;
+    background: none;
+    color: $white;
+    font-weight: 700;
+  }
+
+  &__cvv {
+    flex: 1;
+    display: inline-block;
+    text-align: right;
+  }
+
+  &__cvv-digits {
+    display: inline-block;
+    border: none;
+    width: 30px;
+    text-align: right;
+    background: none;
+    color: $white;
+    font-weight: 700;
+  }
+
+  &__label {
+    text-transform: uppercase;
+    font-size: 0.8em;
+  }
+
+  &__text {
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+
+  &__type {
+    font-weight: 300;
+    color: $white;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    font-size: 0.9em;
+  }
+}
+
+// GRADIENTS
+.jca-credit-card {
+  &--gradient-1 {
+    background: linear-gradient(123deg, #FFFFFF 0%, #00B2FF 100%),
+                linear-gradient(236deg, #BAFF99 0%, #005E64 100%),
+                linear-gradient(180deg, #FFFFFF 0%, #002A5A 100%),
+                linear-gradient(225deg, #0094FF 20%, #BFF4ED 45%, #280F34 45%, #280F34 70%, #FF004E 70%, #E41655 85%, #B30753 85%, #B30753 100%),
+                linear-gradient(135deg, #0E0220 15%, #0E0220 35%, #E40475 35%, #E40475 60%, #48E0E4 60%, #48E0E4 68%, #D7FBF6 68%, #D7FBF6 100%);
+    background-blend-mode: overlay, overlay, overlay, darken, normal;
+  }
+
+  &--gradient-2 {
+    background: linear-gradient(60deg, #2B2E4A 0%, #2B2E4A 30%, #E84545 calc(30% + 1px), #E84545 60%, #903749 calc(60% + 1px), #903749 70%, #53354A calc(70% + 1px), #53354A 100%);
+  }
+
+  &--gradient-3 {
+    background: radial-gradient(100% 225% at 100% 0%, #FF0000 0%, #000000 100%),
+                linear-gradient(236deg, #00C2FF 0%, #000000 100%),
+                linear-gradient(135deg, #CDFFEB 0%, #CDFFEB 36%, #009F9D 36%, #009F9D 60%, #07456F 60%, #07456F 67%, #0F0A3C 67%, #0F0A3C 100%);
+    background-blend-mode: overlay, hard-light, normal;
+  }
+
+  &--gradient-4 {
+    background: linear-gradient(to bottom, rgba(255,255,255,0.15) 0%, rgba(0,0,0,0.15) 100%),
+                radial-gradient(at top center, rgba(255,255,255,0.40) 0%, rgba(0,0,0,0.40) 120%) #989898; 
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5715280)